consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/validation/MessageValidatorFactory.java

```diff
@@ -56,7 +56,7 @@ public RoundChangeMessageValidator createRoundChangeMessageValidator(
     return new RoundChangeMessageValidator(
         roundIdentifier -> createMessageValidator(roundIdentifier, parentHeader),
         validators,
-        IbftHelpers.calculateRequiredValidatorQuorum(validators.size()),
+        IbftHelpers.calculateRequiredValidatorQuorum(validators.size()) - 1,
```

Comment:
Why do you want one less than the required quorum of validators?  [magic number]

Reply:
This calc has been broken out to a helper function which is now used in many places (which is good).
Throughout IBFT, the number of prepare messages is 1 less than the actual 2/3 quorum as the Proposal is deemed to act as a prepare message (thus, there is also a rule that none of the prepare messages may come from the Proposer).